Projects/Home/Sources/Screens/Home.swift

```diff
@@ -11,10 +11,31 @@ import hCore
 import hCoreUI
 import hGraphQL
 
+public class HomeNavigationViewModel: ObservableObject {
+    public init() {}
+
+    @Published public var isSubmitClaimPresented = false
+    @Published public var isHelpCenterPresented = false
+
+    //claim details
+    @Published public var isChatPresented = false
+    @Published public var isDocumentPresented = false
+    @Published public var document: InsuranceTerm? = nil
+
+    //help center
+    @Published public var isPaymentsPresented = false
```

Comment:
Should these (quick actions) be moved inside its own view model in Help Center instead?